external-import/flashpoint/src/flashpoint_connector/config_loader.py

```diff
@@ -0,0 +1,316 @@
+        default=500,
+    )
+    send_to_queue: bool = Field(
+        description="Connector send-to-queue flag. Default to True.",
+        default=True,
+    )
+    send_to_directory: bool = Field(
+        description="Connector send-to-directory flag.",
+        default=False,
+    )
+    send_to_directory_path: str | None = Field(
+        description="Connector send-to-directory path.",
+        default=None,
+    )
+    send_to_directory_retention: int = Field(
+        description="Connector send-to-directory retention.",
+        default=7,
+    )
+
+
+class FlashpointConfig(ConfigBaseModel):
+    """
+    Define config specific to Flashpoint.
+    """
+
+    api_key: str = Field(description="The API key to connect to Flashpoint.")
+    import_start_date: DatetimeFromIsoString = Field(
+        description="The date from which to start importing data.",
+        default_factory=lambda: iso_string_validator("P30D"),  # 30 days ago
+    )
+    import_reports: bool = Field(
+        description="Whether to import reports from Flashpoint or not.",
+        default=True,
+    )
+    indicators_in_reports: bool = Field(
+        description="Whether to include indicators in the reports imported from MispFeed or not.",
+        default=False,
+    )
+    import_indicators: bool = Field(
+        description="WHether to import indicators of compromise (IoCs) or not.",
+        default=True,
+    )
+    import_alerts: bool = Field(
+        description="Whether to import alert data from Flashpoint or not.",
+        default=True,
+    )
+    alert_create_related_entities: bool = Field(
+        description="Whether to create alert related Channel entity and Media-Content observable or not.",
+        default=False,
+    )
+    import_communities: bool = Field(
+        description="Whether to import community data or not.",
+        default=False,
+    )
+    communities_queries: ListFromString = Field(
+        description="List of community queries to execute.",
+        default=["cybersecurity", "cyberattack"],
+    )
+    interval: int = Field(
+        description="The number of seconds to await between two runs of the conector.",
+        deprecated="Config variable 'interval' is deprecated, please use 'duration_period' instead.",
+        default=5,
+    )
+
+
+class ConfigLoader(BaseSettings):
+    """
+    Define a complete config for a connector with:
+        - opencti: the config specific to OpenCTI
+        - connector: the config specific to the `external-import` connectors
+        - flashpoint: the config specific to Flashpoint
+    """
+
+    opencti: OpenCTIConfig
+    connector: ConnectorConfig
+    flashpoint: FlashpointConfig
+
+    # Setup model config and env vars parsing
+    model_config = SettingsConfigDict(
+        frozen=True,
+        extra="allow",
+        env_nested_delimiter="_",
+        env_nested_max_split=1,
+        enable_decoding=False,
+        yaml_file="./src/config.yml",
```

Comment:
```suggestion
        yaml_file="../config.yml",
```
Also, it should be at the root and out of the src folder wdyt ?

Reply:
I mean how could it work with the yml file, knowing the file loads the config from `/src/flashpoint_connector/config_loader.py` and not `.`

I think it tries to get the file here : 
`/src/flashpoint_connector/src/config.yml`

Reply:
debugged and fixed together ✅